tests/CMakeLists.txt

```
@@ -0,0 +1,6 @@
+# heir-check will run the lit based tests (add option -a for lit to show the details of tests and/or failures)
+add_custom_target(heir-check
```

for consistency with MLIR

```suggestion
add_custom_target(check-heir
```